app/src/App.js

```diff
@@ -1,16 +1,37 @@
 import * as React from "react";
 
 // import * as apiClient from "./apiClient";
+import { useAuth0 } from "@auth0/auth0-react";
 import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
 import Day from "./components/Day";
 import Week from "./components/Week";
 import logo from "./moon-logo.svg";
 
 const App = () => {
-  return (
+  const {
+    user,
+    isAuthenticated,
+    isLoading,
+    loginWithRedirect,
+    logout,
+  } = useAuth0();
+
+  if (isLoading) {
```

Comment:
George just showed me another way to refactor this
delete lines 20-21, we'll add that into a ternary.

then we can replace` return isAuthenticated ? (` with:
`return isLoading ? <div>Loading ...</div> : isAuthenticated ? (`